docusaurus/docs/iOS/03-guides/04-noise-cancellation.mdx

```diff
@@ -48,14 +48,22 @@ Once you are able to create a `NoiseCancellationFilter` you can rely on `Call`'s
 - `.available`
 The featue has been enabled on the dashboard and it's available for the call. In this case, you are free to present any noise cancellation toggle UI in your application.
 
+:::important
+Even though the feature may be enabled for your call, you should not that NoiseCancellation is a very performance-heavy process. For that reason, it's recommended to only allow the feature on devices that support Apple's neuralEngine. 
+
+You can easily check if the current device has neuralEngine support by using `StreamVideo.isHardwareAccelerationAvailable` on your streamVideo initialized instanceof.
```

Comment:
the end of the sentence is a bit confusing. Also, we should add a code snippet how to do that.